packages/embedded/src/routes/api/reports.ts

```diff
@@ -0,0 +1,28 @@
+import { Hono } from "hono";
+import { PLAYGROUND_SERVICES_URL } from "@/const.js";
+
+export default function createReportsApiRoute(apiKey: string) {
+  const app = new Hono();
+  
+  // Proxy all requests to fp-services but attach a token
+  app.all("*", async (c) => {
+    const url = `${PLAYGROUND_SERVICES_URL}${c.req.path}`;
+  
+    const contentType = c.req.header("content-type");
+    const headers = new Headers();
+    // Only include the bare minimum authentication and content-type headers
+    headers.set("Authorization", `Bearer ${apiKey}`);
```

Comment:
do we need to do both of these auth methods? (Authorization Bearer and x-fpx-token)

Reply:
removed!